bitrise.yml

```diff
@@ -1463,7 +1463,103 @@ workflows:
             App|${BITRISE_APP_URL}
             #mobile-testeng|https://mozilla.slack.com/archives/C02KDDS9QM9
             Mana|https://mana.mozilla.org/wiki/display/MTE/Mobile+Test+Engineering
-
+  Firebase_Performance_Test:
+    before_run:
+    - 1_git_clone_and_post_clone
+    - 2_certificate_and_profile
+    - 3_provisioning_and_npm_installation
+    steps:
+    - xcode-archive@4.7.2:
```

Comment:
Not sure that you need to archive the app... this step is to generate the IPA file. We need to upload it for the robo tests. However, in order to run tests in a test plan, you only need the zip file with the `xctestrun` and the `Fennec_Enterprise-iphoneos` folder in it

Reply:
left this in there for archiving the env vars as i _think_ they are being passed and used by firebase, but i'm not sure. that's why initially i included all the runtime args in the xcode script command just to be sure. since i removed them, i was leaving them in as a backstop should firebase need them, but i'm really not sure if they have any impact.